code/modules/spells/ability_types/realized.dm

```diff
@@ -31,6 +31,192 @@
+	var/damage_slowdown = 0.2 // Slowdown per pulse
+
+/obj/effect/proc_holder/ability/lamp/Perform(target, mob/user)
+	cooldown = world.time + (2 SECONDS)
+	if(!do_after(user, 1.5 SECONDS))
+		to_chat(user, "<span class='warning'>You must stand still to see!</span>")
+		return
+	playsound(get_turf(user), 'sound/abnormalities/bigbird/hypnosis.ogg', 75, 0, 2)
+	for(var/mob/living/L in view(damage_range, user))
+		if(user.faction_check_mob(L, FALSE))
+			continue
+		if(L.stat == DEAD)
+			continue
+		new /obj/effect/temp_visual/revenant(get_turf(L))
+		if(ishostile(L))
+			var/mob/living/simple_animal/hostile/H = L
+			H.TemporarySpeedChange(damage_slowdown, 15 SECONDS) // Slow down
+			var/new_overlay = mutable_appearance('ModularTegustation/Teguicons/tegu_effects.dmi', "enchanted", -HALO_LAYER)
+			H.add_overlay(new_overlay)
+			addtimer(CALLBACK (H, .atom/proc/cut_overlay, new_overlay), 15 SECONDS)
+			H.apply_status_effect(/datum/status_effect/salvation)
+	return ..()
+
+/datum/status_effect/salvation
+	id = "salvation"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 15 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/salvation
+
+/datum/status_effect/salvation/on_apply()
+	. = ..()
+	var/mob/living/simple_animal/M = owner
+	if(M.damage_coeff[RED_DAMAGE] <= 0)
+		return
+	M.damage_coeff[RED_DAMAGE] *= 1.1
+	if(M.damage_coeff[WHITE_DAMAGE] <= 0)
+		return
+	M.damage_coeff[WHITE_DAMAGE] *= 1.1
+	if(M.damage_coeff[BLACK_DAMAGE] <= 0)
+		return
+	M.damage_coeff[BLACK_DAMAGE] *= 1.1
+	if(M.damage_coeff[PALE_DAMAGE] <= 0)
+		return
+	M.damage_coeff[PALE_DAMAGE] *= 1.1
+
+/datum/status_effect/salvation/on_remove()
+	. = ..()
+	var/mob/living/simple_animal/M = owner
+	M.damage_coeff[RED_DAMAGE] /= 1.1
+	M.damage_coeff[WHITE_DAMAGE] /= 1.1
+	M.damage_coeff[BLACK_DAMAGE] /= 1.1
+	M.damage_coeff[PALE_DAMAGE] /= 1.1
+
+/atom/movable/screen/alert/status_effect/salvation
+	name = "Salvation"
+	desc = "You will be saved."
+	icon = 'icons/mob/actions/actions_ability.dmi'
+	icon_state = "salvation"
+
+/* Nothing There - Shell */
+/obj/effect/proc_holder/ability/goodbye
+	name = "Goodbye"
+	desc = "An ability that does massive damage in an area and heals you."
+	action_icon_state = "goodbye0"
+	base_icon_state = "goodbye"
+	cooldown_time = 30 SECONDS
+
+	var/damage_amount = 500 // Amount of good bye damage
+	var/damage_range = 9
+
+/obj/effect/proc_holder/ability/goodbye/Perform(target, mob/user)
+	var/mob/living/carbon/human/H = user
+	cooldown = world.time + (5 SECONDS)
+	playsound(get_turf(user), 'sound/abnormalities/nothingthere/goodbye_cast.ogg', 75, 0, 5)
+	if(!do_after(user, 4.5 SECONDS))
+		to_chat(user, "<span class='warning'>You must stand still to do the nothing there classic!</span>")
+		return
+	for(var/turf/T in view(2, user))
+		new /obj/effect/temp_visual/smash_effect(T)
+		for(var/mob/living/L in T)
+			if(user.faction_check_mob(L, FALSE))
+				continue
+			if(L.stat == DEAD)
+				continue
+			H.adjustBruteLoss(-10)
```

Comment:
Healing on top of 500 damage in a 5x5 might be too strong. Just keep this in consideration. Not a requested change at this time.

Reply:
will lower the damage to 400